src/hotspot/share/gc/shenandoah/shenandoahMmuTracker.cpp

```diff
@@ -182,9 +265,34 @@ bool ShenandoahMmuTracker::transfer_capacity(ShenandoahGeneration* from, Shenand
   return true;
 }
 
-void ShenandoahMmuTracker::initialize() {
-  _initial_process_time_s = process_time_seconds();
-  _initial_collector_time_s = gc_thread_time_seconds();
-  _initial_verify_collector_time_s = _initial_collector_time_s;
-  _mmu_periodic_task->enroll();
-}
\ No newline at end of file
+size_t round_down_to_multiple_of_region_size(size_t bytes) {
```

Comment:
I could have sworn there was a rounding utility/macro extensively used in sizing code, but the only one I found was a power of 2 rounder. The alternative, if one maintained a log of heap region size (being a power of 2) would be to use a bit-mask here.

Anyway, nothing to do here; this looks good for now.

Reply:
I wasn't sure if region size was coerced to a power of a 2, but it seems to be the case:
```
size_t ShenandoahHeapRegion::setup_sizes(size_t max_heap_size) {
// ...
  int region_size_log = log2i(region_size);
  // Recalculate the region size to make sure it's a power of
  // 2. This means that region_size is the largest power of 2 that's
  // <= what we've calculated so far.
  region_size = size_t(1) << region_size_log;
```
I'll make that change.